### ISSUE TITLE
Package.json type: "module"

### ISSUE THREAD

**Issue by philholden** [jakearchibald/idb-keyval#124]
Had the following build error when using `idb-keyval` with `sveltekit` and `adapter-cloudflare-worker`.

```
(node:51281) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
(Use `node --trace-warnings ...` to show where the warning was created)
> Cannot use import statement outside a module
/Users/holdph/personal/song-kit/node_modules/idb-keyval/dist/esm/index.js:1
import safariFix from 'safari-14-idb-fix';
^^^^^^
```

I added `"type": "module",` to the `package.json` of `node_modules` for both `idb-keyval` and `safari-14-idb-fix` and that allows me to build locally.

**Comment by philholden** [comment]
https://github.com/jakearchibald/safari-14-idb-fix/pull/6
https://github.com/jakearchibald/idb-keyval/pull/125

**Comment by jakearchibald** [comment]
Thank you for this! Fixed version published as 5.1.4.
